docs/trino-connector/development.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Gravitino connector development"
+slug: /trino-connector/development
+keyword: gravitino connector development 
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+This document is to guide users through the development of the Gravitino connector for Trino locally.
+
+## Prerequisites
+
+Before you start developing the Gravitino trino connector, you need to have the following prerequisites:
+
+1. You need to start the Gravitino server locally, for more information, please refer to the [start Gravitino server](../how-to-install.md)
+2. Create a catalog in the Gravitino server, for more information, please refer to the [Gravitino metadata management](../manage-metadata-using-gravitino.md). Assuming we have just created a MySQL catalog using the following command:
+
+```curl
+curl -X POST -H "Content-Type: application/json" -d '{"name":"test","comment":"comment","properties":{}}' http://localhost:8090/api/metalakes
+
+curl -X POST -H "Content-Type: application/json" -d '{"name":"mysql_catalog3","type":"RELATIONAL","comment":"comment","provider":"jdbc-mysql", "properties":{
+  "jdbc-url": "jdbc:mysql://127.0.0.1:3306?useSSL=false&allowPublicKeyRetrieval=true",
+  "jdbc-user": "root",
+  "jdbc-password": "123456",
+  "jdbc-driver": "com.mysql.cj.jdbc.Driver"
+}}' http://localhost:8090/api/metalakes/test/catalogs
+```
```

Comment:
catalogs are innecessary for setting up dev environment.

Reply:
Can you use the Trino command line to display the Gravition catalog if you do not create catalogs here?

Reply:
yes. i just start gravitino server built from source, and add a test empty metalake. on trino side, there is only `gravitino.properties` under the catalog dir, and Trino dev server started up normally.

Reply:
I mean if we could list the catalogs in the format like "test.mysql_catalog" by the Trino client.

Reply:
like 
![image](https://github.com/datastrato/gravitino/assets/15794564/22c20939-31c2-484a-afc2-cc4b326d51c8)